tests/test_tocdirective/test_toc_urllink.html

```diff
@@ -1,6 +1,6 @@
 <div class="tableofcontents-wrapper compound">
 <ul>
-<li class="toctree-l1"><a class="reference internal" href="content1.html">Content1</a></li>
+<li class="toctree-l1"><a class="reference internal" href="content1.html">Content 1</a></li>
```

Comment:
@AakashGfude is it expected that `Content 1` here has a space but none of the others contain a space.

Reply:
@mmcky yes. Because the `_toc.yml` for this file, does not have a `-title` specified for `Content1`, so it takes the title from the doc itself, which is `Content 1`. Did this change in `toc`  to increase coverage of the code.

Reply:
ok - great. Thanks @AakashGfude